camelot/view/action_steps/crud.py

```diff
@@ -108,14 +115,55 @@ class SetColumns(ActionStep):
     
     def __init__(self, static_field_attributes):
         self.static_field_attributes = static_field_attributes
+        self.column_attributes = []
+        for fa in static_field_attributes:
+            attrs = {}
+            if issubclass(fa['delegate'], (delegates.ComboBoxDelegate, delegates.Many2OneDelegate,
+                                           delegates.FileDelegate)):
+                attrs = self.filter_attributes(fa, ['action_routes'])
+            elif issubclass(fa['delegate'], delegates.DateDelegate):
+                attrs = self.filter_attributes(fa, ['nullable', 'validator'])
+                if issubclass(fa['delegate'], delegates.DateTimeDelegate):
```

Comment:
ik denk niet dat we editeerbare datetime velden hebben, dus in principe mag die editor hardcoded not editable zijn ...